src/main.ts

```diff
@@ -3,13 +3,21 @@ import {downloadTool, extractZip} from "@actions/tool-cache";
 import {GitHub} from "@actions/github";
 import {resolve} from "path";
 import {exec} from "@actions/exec";
+import configFile from "./configFile";
 import foreman from "./foreman";
 
 async function run(): Promise<void> {
   try {
     const versionReq: string = getInput("version");
     const githubToken: string = getInput("token");
     const workingDir: string = getInput("working-directory");
+    const allowExternalGithubOrgs: string = getInput(
+      "allow-external-github-orgs"
+    ).toLowerCase();
+
+    if (allowExternalGithubOrgs == null || allowExternalGithubOrgs != "true") {
```

Comment:
I think this is redundant, it should reduce to:
```suggestion
    if (allowExternalGithubOrgs != "true") {
```